task11/index.js

```diff
@@ -0,0 +1,54 @@
+const div = document.getElementById("js-parent");
```

Comment:
MUST
`div` is not meaningful
Let's name it properly
Here we load, display an error and display the content

Reply:
I named it without thinking about it.
From now on, be careful to think about it and give it a name.
Thank you very much.